src/model/src/AERO_PHOTDATA.F

```diff
@@ -2079,25 +2079,25 @@ SUBROUTINE FAST_OPTICS( NR, NI, LAMBDA, DGN, SIGMA_G, BETA_EXT, BETA_SCAT, G )
          Z12   = Z1    * Z1
          Z2    = 4.0   * XNRI2 + 12.0 * XNRMI + 9.0
          XC1   = 8.0   / ( 3.0 * Z12 )
-         A1    = DBLE(24.0  * XRI / Z1) !(Wei Li)
+         A1    = DBLE(24.0  * XRI / Z1)
 
-         A2    = DBLE(44.0   * XRI / 15.0 + 20.0 * XRI / ( 3.0 * Z2 ) +
+         A2    = DBLE(4.0   * XRI / 15.0 + 20.0 * XRI / ( 3.0 * Z2 ) +
```

Comment:
Double check this... its a big change

Reply:
the changed '4' version is correct. double checked the code